FreeRTOS/Test/CMock/message_buffer/message_buffer_utest.c

```diff
@@ -328,25 +361,31 @@ void test_xMessageBufferCreate_malloc_fail( void )
  */
 void test_xMessageBufferCreate_zero_size( void )
 {
-    if( TEST_PROTECT() )
-    {
-        ( void ) xMessageBufferCreate( 0 );
-    }
+    xMessageBuffer = xMessageBufferCreate( 0 );
```

Comment:
This should test for an assertion from the code [here](https://github.com/FreeRTOS/FreeRTOS-Kernel/blob/main/stream_buffer.c#L248) and the test should not continue after that.

Reply:
Asserts can be turned off in production. This tests both the configASSERT disabled or configASSERT enabled.